sympy/physics/mechanics/tests/test_joint.py

```diff
@@ -282,16 +282,15 @@ def test_pinjoint_arbitrary_axis():
     angle = A.ang_vel_in(N).angle_between(A.x+A.y-A.z)
     assert angle.xreplace({omega: 1}) == 0
     assert (C.masscenter.vel(N).simplify() ==
-            (m*omega*N.y + m*omega*N.z + n*omega*A.y + n*omega*A.z)/sqrt(3))
+            sqrt(3)*n*omega/3*A.y + sqrt(3)*n*omega/3*A.z)
```

Comment:
Did you work this out by hand to verify it?

Reply:
I have used a few methods, since the answer was a bit different than I expected. I used the following code which reuses the defined ReferenceFrames:
```py
Pm = Point("Pm")
Pm.set_vel(N, 0)
Pj = Pm.locatenew('Pj', m * N.x)
Pj.set_vel(N, 0)
Pj.set_vel(A, 0)
Cm = Pj.locatenew('Cj', - n * A.x)
Cm.set_vel(A, 0)
print(Cm.v2pt_theory(Pj, N, A).simplify())
```
Besides this I also plotted the situation with which you can see, that the velocity `n*omega` is split in three parts, of which only the `y` and `z` component appear in the velocity since the `x` is parallel to the arm. I do also expect that the last check is a logical change, since it now checks if indeed the velocity is perpendicular to the axis of the PinJoint.